### PR TITLE
Correct the sticky positioning of the header on mobile screen widths.

### PR DESCRIPTION
**Pull request by benbowler**
## Summary

Addresses issue #2331

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/sass/components/global/_googlesitekit-header.scss [hunk 1]
```diff
@@ -21,20 +21,20 @@
...
-		top: 46px;
-	}
-
 	@media (min-width: $bp-wpAdminBarTablet) {
 		top: 32px;
 	}
 
+	@media (min-width: $bp-tablet) {
+		top: 46px;
+	}
```
Comment (aaemnnosttv) [inline assets/sass/components/global/_googlesitekit-header.scss:36]: This is the correct rule but should come before the previous media query as it is currently resulting in a gap in larger viewports.
image

image

**Review by aaemnnosttv — CHANGES_REQUESTED** [review]
This is almost good to go @benbowler – it seems to just need a tweak to the ordering of the media queries – please double-check that though 👍

File: assets/sass/components/global/_googlesitekit-header.scss (hunk 1 above)
Comment (benbowler) [reply assets/sass/components/global/_googlesitekit-header.scss:36]: Good spot.

**Review by aaemnnosttv — APPROVED** [review]
Super, thanks!

**Pull request merged**